src/utils/client.ts

```diff
@@ -574,11 +574,31 @@ export default class Client {
     );
   }
 
-  createPayment(paymentMethodFieldKey: string) {
-    return this.payment('POST', { field_id: paymentMethodFieldKey });
+  createPayment() {
+    return this.payment('POST');
   }
 
   paymentComplete() {
     return this.payment('PUT');
   }
+
+  async createCheckoutSession(successUrl: string, cancelUrl?: string) {
+    await initFormsPromise;
+    const { userId } = initInfo();
+    const url = `${API_URL}stripe/checkout/`;
+    const data = {
+      form_key: this.formKey,
+      ...(userId ? { user_id: userId } : {}),
```

Comment:
i think you may be able to assume userId always exists, i don't think we'd support payment actions without knowing who the user is.